scope_ascom.cpp

```diff
@@ -190,11 +190,11 @@ void ScopeASCOM::SetupDialog()
             wxMessageBox(msg, _("Error"), wxOK | wxICON_ERROR);
         }
     }
-    // destroy the COM object now as this reduces the likelhood of getting into a
-    // state where the user has killed the ASCOM local server instance and PHD2 is
-    // holding a reference to the defunct driver instance in the global interface
-    // table

```

Comment:
it's good that we added the new comment explaining why the de-registration is conditional, but I think we should also keep this comment explaining why we do the unresgtation